code/src/cljs/sixsq/nuvla/ui/edges/views.cljs

```diff
@@ -732,8 +732,9 @@
 
 
 (defn NuvlaboxRow
-  [{:keys [id name description created state tags online] :as _nuvlabox} managers]
-  (let [uuid (general-utils/id->uuid id)]
+  [{:keys [id name description created state tags online] :as _nuvlabox} managers next-heartbeats]
+  (let [uuid (general-utils/id->uuid id)
+        next-heartbeat (get-in @next-heartbeats [id :next-heartbeat])]
```

Comment:
this can be a reactive subscription to avoid passing it down. will show you how we do that